windows-esd-to-iso

```diff
@@ -1,30 +1,31 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
-set -ex
+set -Eeuo pipefail -o posix
 
 esd="${1}"
 tmpdir=$(mktemp -d)
 
+if command -v brew &>/dev/null; then
+  brew install wimlib
+fi
```

Comment:
I do not believe in scripts changing the user's system. I would support checking to see if wimlib tools are available and giving the user a useful message explaining that they need to be installed, directing them to the dependencies section of the README.